alpenhorn/storage.py

```diff
@@ -374,3 +374,60 @@ def update_avail_gb(self, new_avail: int | None) -> None:
 
         if new_avail is None:
             log.info(f'Unable to determine available space for "{self.name}".')
+
+
+class StorageTransfer(base_model):
```

Comment:
As I think about it more and more I think the naming for this is awkward. To me it sounds like it should represent a single transfer, i.e. a file being moved etc. In reality it represents configuration for what actions should be triggered when a file is moved. Maybe `StorageTransferRule`? `StorageTransferAction`?

For the future, I think if we want configuration for the actual transfers along specific edges, it's probably clearest to put them in a separate table.